Bash_scripts/HW21/in_hw20/group_vars/centos/centos_var.yml

```diff
@@ -0,0 +1,4 @@
+#Centos vars
+ansible_user: vagrant
```

Comment:
Do you need this var here if you already have it under `linux` group?

Reply:
Sorry, this is not the job, this is hw21, everything is fixed, the directory is deleted